velocity-custom-parser-example/pom.xml

```diff
@@ -0,0 +1,267 @@
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.velocity</groupId>
+        <artifactId>velocity-engine-parent</artifactId>
+        <version>2.2-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>velocity-custom-parser-example</artifactId>
+    <name>Apache Velocity Custom Parser - Example</name>
+    <description>Custom Parser Example for Apache Velocity</description>
+
+    <packaging>jar</packaging>
+
+    <!--
+      This plugin demonstrates how to build a custom Velocity parser.
+      The proposed custom parser replaces '#' with '@' and '@' with '%',
+      so that it's suitable to use with Markdown template files, for instance.
+
+      The generated parser class is org.apache.velocity.runtime.parser.${parser.name}Parser,
+      and must be specified at runtime using the Velocity property parser.class :
+        parser.class = org.apache.velocity.runtime.parser.MyCustomParser
+    -->
+
+    <properties>
+        <!-- whether to display debug logs while parsing -->
+        <parser.debug>false</parser.debug>
+        <!-- parser name -->
+        <parser.name>Custom</parser.name>
+        <!-- character to substitute to '*' -->
+        <parser.char.asterisk>*</parser.char.asterisk>
+        <!-- character to substitute to '@' -->
+        <parser.char.at>%</parser.char.at>
+        <!-- character to substitute to '$' -->
+        <parser.char.dollar>$</parser.char.dollar>
+        <!-- character to substitute to '#' -->
+        <parser.char.hash>@</parser.char.hash>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.velocity</groupId>
+            <artifactId>velocity-engine-core</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <version>${slf4j.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.6</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <defaultGoal>install</defaultGoal>
+        <plugins>
+            <!-- generate manifest file -->
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+            </plugin>
+            <!-- extract raw parser grammar from velocity jar -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>3.1.1</version>
```

Comment:
This should be also in the parent. AFAIR, I have added all important plugins to the parents.